app/src/main/java/com/duycuong/weather/ui/screen/main/MainActivity.java

```diff
@@ -0,0 +1,128 @@
+package com.duycuong.weather.ui.screen.main;
+
+import android.Manifest;
+import android.annotation.SuppressLint;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
```

Comment:
ERROR: Unused import: android.content.DialogInterface. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports